addons/account/static/src/js/calculator_widget.js

```diff
@@ -0,0 +1,62 @@
+odoo.define('account.calculator', function (require) {
+"use strict";
+
+var basic_fields = require('web.basic_fields');
+var field_registry = require('web.field_registry');
+var field_utils = require('web.field_utils');
+
+var FieldCalculator = basic_fields.FieldMonetary.extend({
+    className: 'o_field_monetary o_field_number',
+    tagName: 'span',
+    supportedFieldTypes: ['float', 'monetary'],
+    resetOnAnyFieldChange: true, // Have to listen to currency changes
+    /**
+     * Evaluate value set by user if starts with =
+     * Only allows floats and + - * / ()
+     *
+     * @override
+     * @private
+     * @param {any} value
+     * @param {Object} [options]
+     */
+    _setValue: function (value, options) {
+        var val = value.trim();
+        if (val.startsWith('=')) {
+            try {
+                // remove = and extra space
+                var val = val.replace(new RegExp(/([ =])/g), '');
+                // split on +-*/() and keep them in the result split_array
+                // example: a = '2+2' -> a.split(regex) = ['2', '+', '2']
+                var split_vals = val.split(new RegExp(/([-+*/()])/g));
+                var safe_eval_string = '';
+                $.each(split_vals, function(index, value) {
+                    if (!['+','-','*','/','(',')'].includes(value) && value.length) {
```

Comment:
It should be nice accept `^`

Reply:
Usefull when computing areas and volumes in sale orders when selling those kind of products with unint of measure in Area and VOlumes.

Reply:
@nhomar you want bitwise XOR? I think you mean `**`

Reply:
Also it would be nice to accept `√`  😁

Reply:
@mreficent you can do it using ^0.5 (**0.5) which is the same as √ ;)